src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs

```diff
@@ -214,13 +234,9 @@ private void ScheduleTimer()
                     // If total retries are exhausted, schedule the call back for next interval mentioned in Settings.xml.
                     if (this.IncrementRetryCount() == false)
                     {
-                        this._nodeAgentSfUtility.ReportHealth("WindowsUpdateOperationResult",
-                            "Reschedule retries exhausted. Resetting the state machine.", HealthState.Warning, -1,
-                            TimeSpan.FromMinutes(this._serviceSettings.OperationTimeOutInMinutes));
                         if (this._windowsUpdateManager.ResetStateMachine())
                         {
-                            this.UpdateSettingsAndCreateCheckpoint();                            
-                            this.CreateNewCheckpointFile();                            
+                            this.UpdateSettingsAndCreateCheckpoint();
                         }
```

Comment:
Why removing this?

Reply:
Because UpdateSettingsAndCreateCheckPoint also creates new checkpoint